lib/include/jxl/encode.h

```diff
@@ -335,6 +335,12 @@ typedef enum {
    */
   JXL_ENC_FRAME_SETTING_JPEG_COMPRESS_BOXES = 33,
 
+  /** Control what kind of buffering is used, when using chunked image frames.
+   * 0 = can buffer anything, 1 = can buffer intermediate data,
+   * 2 = can buffer the output, 3 = minimize buffer usage
+   */
+  JXL_ENC_FRAME_SETTING_BUFFERING = 34,
```

Comment:
It's not clear to me what the option values mean, and what the implications of this are on how the chunked encode api can/should be used and/or how it affects compression.

Reply:
Added more explanation